pkg/experiment/compactor/compaction_worker.go

```diff
@@ -50,11 +50,12 @@ type Worker struct {
 }
 
 type Config struct {
-	JobConcurrency  int           `yaml:"job_capacity"`
-	JobPollInterval time.Duration `yaml:"job_poll_interval"`
-	SmallObjectSize int           `yaml:"small_object_size_bytes"`
-	TempDir         string        `yaml:"temp_dir"`
-	RequestTimeout  time.Duration `yaml:"request_timeout"`
+	JobConcurrency        int           `yaml:"job_capacity"`
+	JobPollInterval       time.Duration `yaml:"job_poll_interval"`
+	SmallObjectSize       int           `yaml:"small_object_size_bytes"`
+	TempDir               string        `yaml:"temp_dir"`
+	RequestTimeout        time.Duration `yaml:"request_timeout"`
+	EnableMetricsExporter bool          `yaml:"enable_metrics_exporter"`
```

Comment:
For consistency it should be

```suggestion
	MetricsExporterEnabled bool          `yaml:"metrics_exporter_enabled"`
```

Eventually it might get more than just this one setting and then a separate struct might make sense, with only Enabled as config option.